### ISSUE TITLE
check new logging options in openldap 2.6

### ISSUE THREAD

**Issue by davidcoutadeur** [ltb-project/openldap-deb#69]

**Comment by davidcoutadeur** [comment]
What's working in 2.5.7 is quite similar to 2.4 logging features:
- directive logfile creates a file (this directive was also present in 2.4)
- when OpenLDAP runs with -d (debug option, stay attached), then the logs sent to stderr are also sent to the logfile
- the main logging system remains syslog

Sources:
- https://bugs.openldap.org/show_bug.cgi?id=6659
- https://bugs.openldap.org/show_bug.cgi?id=6949
- https://git.openldap.org/openldap/openldap/-/merge_requests/358

Seems there is work for a global refactoring of logging system. Wait and see in 2.6 / 3.0 OpenLDAP version

**Comment by davidcoutadeur** [comment]
new 2.6 version relies on logfile directive
See ITS#6949 https://bugs.openldap.org/show_bug.cgi?id=6949

**Comment by davidcoutadeur** [comment]
For information, see this config example: (taken from https://bugs.openldap.org/show_bug.cgi?id=6949)

```
ldapmodify -x -H ldapi:/// -D cn=config -w secret
dn: cn=config
changetype: modify
add: olcLogFile
olcLogFile: /var/symas/slapd.log
-
add: olcLogFileOnly
olcLogFileOnly: TRUE
-
add: olcLogFileRotate
olcLogFileRotate: 12 10 1
```

**Comment by davidcoutadeur** [comment]
Integrated in slapd-cli 2.9, included in openldap LTB 2.6.2 release
